data-raw/prepare_raw_data.R

```diff
@@ -0,0 +1,38 @@
+library(tidyverse)
+library(xml2)
+library(rvest)
+
+# 1. Median Country Ages -------------------------------------------------------
+
+# Decide if we trust worldometers ages
```

Comment:
Just highlighting this - we probably don't want to push a comment like this into working code!